config.template.js

```diff
@@ -17,11 +17,19 @@ config.source_tables = {};
 config.account.api_key = '8675309';
 
 //The name of the user to login as
+//Use username and password if you're
+//not using an access token to authorize this
+//microservice. 
 config.account.username = 'user@user.com';
 
 //The password of the user to login as
 config.account.password = 'correcthorsebatterystaple';
```

Comment:
Yes it is. And if you're not getting the reference, we'll need to talk: https://xkcd.com/936/